class-8/error-handling.js

```diff
@@ -45,5 +45,40 @@ console.log("Welcome to Class-8");
 // const encodedURI = encodeURI("\uD800");
```

Comment:
Uncomment the coding part, just keep the actual comments.

Reply:
Ok , I have done it.